hyperspy/learn/mva.py

```diff
@@ -1072,8 +1071,7 @@ def normalize_poissonian_noise(self, navigation_mask=None,
     def undo_treatments(self):
         """Undo normalize_poissonian_noise"""
         print "Undoing data pre-treatments"
-        self.data = self._data_before_treatments
-        del self._data_before_treatments
+        self.data[:] = self._data_before_treatments
```

Comment:
Now a reference to the original data will be kept indefinitely since we do not `del` it. Won't this lead to possibly not releasing outdated data?